dmaws/utils.py

```diff
@@ -79,6 +79,9 @@ def __init__(self, mapping, variables, **kwargs):
         self._cache = {}
         self._variables = merge_dicts(variables, kwargs)
 
+    def items(self):
```

Comment:
I like a mix of the two actually. I like having an explicit `keys` method and I like the list comprehension based `items` method that knows nothing about `_cache`. Looking at the pull requests as a whole #27 is less contentious and should probably therefore be merged first. Would you object if I changed `items` to the list comprehension style after it's merged?